CONTRIBUTING.md

```diff
@@ -99,4 +99,9 @@ Syntax:
 
 And in case we didn't emphasize it enough: we love tests!
 
+## Writing Tests
+
+* We do write tests for our warns and assertion messages. You may use the `assert.expectAssertion()` and `assert.expectWarning()` helpers for this. Note: Our Travis tests that will run on your PR run in the `production` envrionment, where warnings and assertions are stripped. If you're writing a test for these, you should use `testInDebug` helper rather than `test` to define your test.
+
```

Comment:
Typo: `envrionment` -> `environment`

I think it would be good to add a small sample which clarifies where to import that `testInDebug` helper from and how it is used instead of `test`...